kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkConnector.java

```diff
@@ -108,50 +107,25 @@ private SchemaManager getSchemaManager(BigQuery bigQuery) {
     return new SchemaManager(schemaRetriever, schemaConverter, bigQuery, kafkaKeyFieldName, kafkaDataFieldName);
   }
 
-  private void ensureExistingTables(
-      BigQuery bigQuery,
-      SchemaManager schemaManager,
-      Map<String, TableId> topicsToTableIds) {
-    for (Map.Entry<String, TableId> topicToTableId : topicsToTableIds.entrySet()) {
-      String topic = topicToTableId.getKey();
-      TableId tableId = topicToTableId.getValue();
-      if (bigQuery.getTable(tableId) == null) {
-        logger.info("Table {} does not exist; attempting to create", tableId);
-        schemaManager.createTable(tableId, topic);
-      }
-    }
-  }
-
-  private void ensureExistingTables(
-      BigQuery bigQuery,
-      Map<String, TableId> topicsToTableIds) {
+  private void ensureExistingTables() {
```

Comment:
Since now we are doing lazy table creation. Function name here become kind of confusing. 
When "autoCreateTables" is set to True, `ensureExistingTables()` won't fail, but table actually not create/exists.
Suggest add condition when calling `ensureExistingTables()`, like
```
if (!autoCreateTables) {
    ensureExistingTables()
}
```

Reply:
> Since now we are doing lazy table creation. Function name here become kind of confusing.
> When "autoCreateTables" is set to True, `ensureExistingTables()` won't fail, but table actually not create/exists.
> Suggest add condition when calling `ensureExistingTables()`, like
> 
> ```
> if (!autoCreateTables) {
>     ensureExistingTables()
> }
> ```

Good point, updated accordingly!